src/cljs/main/broadfcui/common/table.cljs

```diff
@@ -399,24 +400,25 @@
      (:query-params @state))
    :update-query-params
    (fn [{:keys [state]} new-params]
-     (swap! state update :query-params merge new-params))
+     (let [change? (not= new-params (select-keys (:query-params @state) (keys new-params)))]
+       (swap! state update :query-params merge new-params)
+       change?))
```

Comment:
It does.  But I see how it looks like it doesn't.  Let me see if I can rewrite that to be clearer.